integration/securitypolicy_test.go

```diff
@@ -27,6 +27,11 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
+var (
```

Comment:
these should be const

Reply:
Yes but the returned value from `fmt.Errorf` or `errors.New` are not feasible to assign constant values.

Reply:
Updated [cb14d3b](https://github.com/TykTechnologies/tyk-operator/pull/625/commits/cb14d3bd5a47e5a6d7ec7938be244525cbc4652a)